libraries/aws_efs_file_systems.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsEfsFileSystems < AwsResourceBase
+  name 'aws_efs_file_systems'
+  desc 'Verifies settings for a collection of AWS EFS File Systems'
+  example "
+    describe aws_efs_file_systems do
+      its('count') { should eq 3 }
+    end
+
+    # Iterate through all file systems
+    aws_efs_file_systems.file_system_ids.each do |file_system_id|
+      describe aws_efs_file_system(file_system_id) do
```

Comment:
similar to a previous comment. Not sure that this is a valid example. The `file_system_id: ` flag should probably be provided

Reply:
Add a note explaining default behavior for string parameters.
6ad5abc